src/test/java/com/thebois/models/world/structures/HouseTests.java

```diff
@@ -42,7 +45,7 @@ public void getTypeEqualsHouse() {
     }
 
     @Test
-    public void testEqualsHouse() {
+    public void areHousesEqual() {
```

Comment:
style

Shouldn't be a question, should be an assertion, e.g. `getTypeReturnsHouse`.